### PR TITLE
React spinner updates and Jest dependency fix

### PR DESCRIPTION
**Pull request by kevinhaube**
This PR updates our Spinner styling and placement, **and fixes a Jest dependency issue!**

Test Steps:
For Jest fix:
1. Run `npm start`.
2. If you receive a message about Jest 26.x.x being a dependency of `react-scripts`, I have failed. If not, you're welcome.

For spinners:
1. Login
2. Navigate to Daily Data
3. You should see a spinner while the table data loads and "Loading Info..." where the org name is until they're loaded.

## Changes
- Removed Jest 27.2.X `devDependency` from `package.json` since 26.x.x is a peer dependency of react-scripts already
- Removed `<Suspense>` from `App.tsx`
- Added `<Spinner>` component
- Used `<Suspense>` on `OrgName` and `TableData` in `Daily.tsx` to show loading indicators

### Testing
- [X] Tested locally?
- [X] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] Downloaded a file from `http://localhost:7071/api/download`?
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Fixes
- #2044

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
No
- Does this PR include changes in authentication and/or authorization of existing endpoints?
No
- Does this change introduce new dependencies that need vetting?
No
- Does this change require changes to our infrastructure?
No
- Does logging contain sensitive data?
No
- Does this PR include or remove any sensitive information itself?
No

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

**Review by Bisonburger — APPROVED** [review]
lgtm

File: frontend-react/src/App.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { Route, useHistory, Switch } from "react-router-dom";
 import { OktaAuth, toRelativeUrl } from "@okta/okta-auth-js";
 import { Security, SecureRoute, LoginCallback } from "@okta/okta-react";
 import { NetworkErrorBoundary } from "rest-hooks";
-import { SpinnerCircular } from "spinners-react";
+import Spinner from './components/Spinner'
```
Comment (TomNUSDS) [inline frontend-react/src/App.tsx:17]: Seems to be unused in this file?

Also the `import { Suspense } from "react";` from line 11 is now unused?

File: frontend-react/src/components/Spinner.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { SpinnerCircular } from 'spinners-react'
+
+function Spinner() {
+    return (
+        <div id="spinner" className="grid-container">
+            <SpinnerCircular
+                color="rgba(57, 88, 172, 1)"
```
Comment (TomNUSDS) [inline frontend-react/src/components/Spinner.tsx:8]: Probably better to have color come out of the scss them via a `className`, but this component doesn't look like it works like that. Lots of other knobs we can turn at some point, too like `secondaryColor`, `speed`, `thickness`.
So, no change really required here.

**Review by TomNUSDS — COMMENTED** [review]
LGTM. The smaller spinner looks much better. There were a couple of unused `import`s that can be removed.

**Pull request merged**